src/main/java/org/opentripplanner/model/TimetableSnapshot.java

```diff
@@ -228,13 +228,12 @@ public Result<?, UpdateError> update(
       FeedScopedId tripId = updatedTripTimes.getTrip().getId();
       TripIdAndServiceDate tripIdAndServiceDate = new TripIdAndServiceDate(tripId, serviceDate);
       realtimeAddedTripPattern.put(tripIdAndServiceDate, pattern);
-    }
 
-    // To make these trip patterns visible for departureRow searches.
-    addPatternToIndex(pattern);
+      // To make these trip patterns visible for departureRow searches.
+      addPatternToIndex(pattern);
```

Comment:
We can remove the check to `pattern.isCreatedByRealtimeUpdater()` inside `addPatternToIndex `